core/src/main/java/com/datastrato/gravitino/catalog/CatalogOperationDispatcher.java

```diff
@@ -546,6 +551,8 @@ public Table alterTable(NameIdentifier ident, TableChange... changes)
                         t.loadTable(NameIdentifier.of(ident.namespace(), tempAlteredTable.name()))),
             NoSuchTableException.class);
 
+    LOG.info("yuqiabc: alterTable: ident: {}, alteredTable: {}", ident, alteredTable.auditInfo());
```

Comment:
typo..

Reply:
Sorry, this PR is still in draft form...